### ISSUE TITLE
Keeping results for major version

### ISSUE THREAD

**Issue by aTrotier** [MagneticResonanceImaging/MRIRecoBenchmarks#7]
Can you run and keep some numerical results for release version (or major changes) ?

**Comment by tknopp** [comment]
Yes, in the long run this would certainly nice. But I creating the infrastructure for something like that seems to be quite complicated. https://github.com/JuliaCI/PkgBenchmark.jl seems to be the way to go for that.

**Comment by aTrotier** [comment]
Maybe we can just write results in a readme ?

Le mar. 11 oct. 2022 à 13:00, Tobias Knopp ***@***.***> a
écrit :
